client/src/main/java/org/apache/rocketmq/client/impl/consumer/ProcessQueue.java

```diff
@@ -102,6 +98,10 @@ public void cleanExpiredMsg(DefaultMQPushConsumer pushConsumer) {
                 log.error("getExpiredMsg exception", e);
             }
 
+            if (msg == null) {
+                break;
```

Comment:
The rest of msgTreeMap's messages can't be processed after the loop terminates,

Is it OK to skip processing message that are currently empty?

Reply:
I think it should be ok.   

in most cases , the rest of msgTreeMap should be empty or interrupted ,in these cases , break the loop is reasonable.

if other cases ,the rest items can wait until the next clean